docs/README.md

```diff
@@ -92,7 +92,7 @@ for more information about installing and using the source code.
 | [Data Column Component](core/components/data-column.component.md) | Defines column properties for DataTable, Tasklist, Document List and other components. | [Source](../lib/core/data-column/data-column.component.ts) |
```

Comment:
not sure those links are ok did you run npm run docbuild?

Reply:
I just did and everything seems fine

Reply:
ok thanks